core/files/fileobject/service.go

```diff
@@ -176,6 +169,12 @@ func (s *service) createInSpace(ctx context.Context, space clientspace.Space, re
 		}
 	}
 
+	if req.AdditionalDetails != nil {
```

Comment:
We should prioritize custom details. For example, relation Name is auto-generated (`fdasfds3244fdsafds.jpg`), but we want to use original name from imported archive (`My awesome picture`)